components/feature/webnotifications/src/main/res/values/strings.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- This Source Code Form is subject to the terms of the Mozilla Public
+   - License, v. 2.0. If a copy of the MPL was not distributed with this file,
+   - You can obtain one at http://mozilla.org/MPL/2.0/.  -->
+<resources>
+    <!-- Default Web Notification Group Name. -->
+    <string name="mozac_feature_notification_group_name">Sites</string>
```

Comment:
How do I get the official strings?

Reply:
Let's punt on this for now since we may want to group site permissions together within the SitePermissionsFeature.

Reply:
@m.heubusch  would be the right person to ping though.

Reply:
Created issue.  https://github.com/mozilla-mobile/fenix/issues/6303.